r/R/schema.R

```diff
@@ -50,6 +50,34 @@
 #' - `$metadata`: returns the key-value metadata as a named list.
 #'    Modify or replace by assigning in (`sch$metadata <- new_metadata`).
 #'    All list elements are coerced to string.
+#'
+#' @section Metadata:
+#'
+#'   Attributes from the `data.frame` are saved alongside tables so that the
+#'   object can be reconstructed faithfully in R (e.g. with `as.data.frame()`).
+#'   This metadata can be both at the top-level of the `data.frame` (e.g.
+#'   `attributes(df)`) or at the column (e.g. `attributes(df$col_a)`) or element
+#'   level (e.g. `attributes(df[1, "col_a"])`). For example, this allows for
```

Comment:
According to the code, this is only true for list columns (which makes sense because regular vectors can't have attributes on elements)